fontir/src/ir.rs

```diff
@@ -98,6 +95,75 @@ pub struct MiscMetadata {
     pub created: Option<DateTime<Utc>>,
 }
 
+/// The name of every glyph, in the order it will be emitted
+///
+/// <https://rsheeter.github.io/font101/#glyph-ids-and-the-cmap-table>
+#[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq, Eq)]
+#[serde(from = "GlyphOrderSerdeRepr", into = "GlyphOrderSerdeRepr")]
```

Comment:
I'm surprised this needs a custom serde impl? It feels simple enough to Just Work.

Also this could potentially be shared with fea-rs at some point, although fea-rs also has to support CID identifiers for CFF, e.g. sometimes we don't get a glyph name, just an integer CID.

Reply:
Having a shared glyph identifier would be nice.

Reply:
I think this is unnecessary indeed, deleted.